chemclipse/plugins/org.eclipse.chemclipse.chromatogram.xxd.peak.detector.supplier.firstderivative/src/org/eclipse/chemclipse/chromatogram/xxd/peak/detector/supplier/firstderivative/settings/FirstDerivativePeakDetectorSettings.java

```diff
@@ -36,52 +37,20 @@
 	@JsonPropertyDescription(value = "Window Size: 3, 5, 7, ..., 45")
 	@EnumSelectionSettingProperty
 	private WindowSize windowSize = WindowSize.WIDTH_5;
+	@JsonProperty(value = "IonFilter", defaultValue = "EXCLUDE")
+	@EnumSelectionRadioButtonsSettingProperty
+	FilterMode filterMode = FilterMode.EXCLUDE;
+	@JsonPropertyDescription(value = "Ions to filter: 16, 18, ...")
+	@StringSettingsProperty
```

Comment:
It doesen't make sense to specify this annotation without any parameter (either multiline or regex). Beside this you could use IMarkedIons as is, no need to convert it to a String.

Reply:
I understand that you can inject, for example, enum types directly via a combo box etc. But for MarkedIons, we do not have an appropriate input widget. So I prove the user with a text field, from which I parse Numbers/ Ions afterwards.

Reply:
See https://github.com/eclipse/chemclipse/issues/41.